### PR TITLE
Feature/error/30

### LINKED ISSUE #30: Return error in JSON format
Return Errors message in json format.

estimate : 30 minutes

### PR DESCRIPTION
**Pull request by JessyBros**
Return error in json format 

1 - update route id -> only number is accepted or route not found
2- In list phones and users, if once doesn't exist return error.

### CODE REVIEW

File: src/Controller/PhoneController.php [hunk 1]
```diff
@@ -23,17 +23,24 @@ class PhoneController extends AbstractController
      */
     public function listPhones(PhoneRepository $phoneRepository, Request $request, PaginatorInterface $paginator): Response
     {
+        if (!$phoneRepository->findAll()) {
```
Comment (crazymad) [inline src/Controller/PhoneController.php:26]: refactor

File: src/Controller/UserController.php [hunk 2]
```diff
@@ -26,6 +26,13 @@ class UserController extends AbstractController
      */
     public function listUsers(UserRepository $userRepository, UserInterface $customer,Request $request, PaginatorInterface $paginator): Response
     {
+        if ($userRepository->findBy(["customer" => $customer])) {
```
Comment (crazymad) [inline src/Controller/UserController.php:29]: refactor

**Review by crazymad — CHANGES_REQUESTED** [review]
404, 403, 500
Validation with Validator

Listener or Subscriber

All responses in JSON

**Comment by JessyBros** [comment]
Close this pull request.
Too early to manage errors. Do this at the end

**Pull request closed without merging**